components/lock.rst

```diff
@@ -431,6 +475,30 @@ method uses the Memcached's ``flush()`` method which purges and removes everythi
     The method ``flush()`` must not be called, or locks should be stored in a
     dedicated Memcached service away from Cache.
 
+PdoStore
+~~~~~~~~~~
+
+Pdo stores rely on the `ACID`_ properties of the SQL engine.
+
+.. caution::
+
+    In a cluster configured with multiple master, ensure writes are
+    synchronously propaged to every nodes, or always use the same node.
+
+.. caution::
+
+    Some SQL engine like MySQL allows to disable unique constraint check.
+    Ensure that this is not the case ``SET unique_checks=1;``.
+
+In order to purge old lock, this store use a current datetime to define a
+expiration date reference. This mechanism relies on all client and server nodes
+to have synchronized clock.
```

Comment:
I read the code PR, is a synchronized clock still needed?

Reply:
Good catch! indeed a previous version uses the client's clock. But it has been replaced by SGDB's  time. 

I removed the `client` from the sentence, as this is still the case for multi-master nodes